data/frame_json.go

```diff
@@ -22,6 +22,7 @@ const simpleTypeString = "string"
 const simpleTypeNumber = "number"
 const simpleTypeBool = "boolean"
 const simpleTypeTime = "time"
+const simpleTypeJSON = "json"
```

Comment:
```suggestion
const simpleTypeOther = "other"
```
The frontend already uses this "anything" bucket:  https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/dataFrame.ts#L19

Reply:
added `simpleTypeOther` in [38bc864](https://github.com/grafana/grafana-plugin-sdk-go/pull/495/commits/38bc864411f73b4588573969c0308311f45de827)